packages/core/src/v3/waitUntil/index.ts

```
@@ -0,0 +1,54 @@
+  }
+
+  setGlobalManager(manager: WaitUntilManager): boolean {
+    return registerGlobal(API_NAME, manager);
+  }
+
+  #getManager(): WaitUntilManager {
+    return getGlobal(API_NAME) ?? NOOP_MANAGER;
+  }
+
+  register(promise: MaybeDeferredPromise): void {
+    return this.#getManager().register(promise);
```

Biome

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)